cloud/blockstore/tools/csi_driver/internal/driver/node_test.go

```diff
@@ -411,3 +412,111 @@ func TestPublishUnpublishDeviceForInfrakuber(t *testing.T) {
 	})
 	require.NoError(t, err)
 }
+
+func TestGetVolumeStatCapabilitiesInVmMode(t *testing.T) {
+	tempDir := os.TempDir()
+
+	nbsClient := mocks.NewNbsClientMock()
+	mounter := mounter.NewMock()
+
+	podID := "test-pod-id-13"
+	diskID := "test-disk-id-42"
+	socketsDir := filepath.Join(tempDir, "sockets")
+	targetPath := filepath.Join(tempDir, "pods", podID, "volumes", diskID, "mount")
+	targetFsPathPattern := filepath.Join(tempDir, "pods/([a-z0-9-]+)/volumes/([a-z0-9-]+)/mount")
+
+	nodeService := newNodeService(
+		"testNodeId",
+		"testClientId",
+		true, // vmMode
```

Comment:
The comments are not needed. Either use variables or remove them completely (I would remove them)

Reply:
removed